### ISSUE TITLE
Some text breaks when copied

### ISSUE THREAD

**Issue by StripedMonkey** [moezbhatti/qksms#948]
```
If I do not make it back in time for lunch, please practice pipetting. I will also send you a list of problems for the dilution formula. Do not stress if you do not get it, as I have had to delay the study guide in order to figure out normality. 

The formula is as follows:
C1 • V1 = C2 • V2

Where:
C1 stands for the molar concentration of the first solution
V1 stands for the volume, in the unit liters, of the first solution
C2 is the concentration of the diluted solution 
V2 stands for the volume of the diluted solution

```
 This fails to copy in qksms V3 B7 but copies without a problem in the default Messenger.

**Comment by MyNameIsCosmo** [comment]
I cannot replicate this issue.

I've copied and pasted from the input box, after sending a message, and after receiving a message.
I've pasted each instance into a notepad, with the expected format.

Could you expand on the issue you're having?

**Comment by StripedMonkey** [comment]
Hmm, well I'm not sure why you're not having the issue but basically I was sent this and wanted to put it into an email. I long press on the message and hit copy and it says "copied". Attempting to paste it anywhere does nothing. I press "paste" and the button just does it's little animation but doesn't disappear and nothing is pasted. Rebooting and restarting do nothing and even now it's still uncopiable in qksms. 

Perhaps there is some character that the default messanger is ignoring or not copying? I'm not sure.

**Comment by moezbhatti** [comment]
I'm able to copy the text without issue - which phone and android version are you using?

**Comment by StripedMonkey** [comment]
Google pixel and Android 8.1

**Comment by moezbhatti** [comment]
Very strange - I'm running the exact same setup.

Could you send this text as a message to yourself and verify that the bug still occurs on the latest version of the app?

**Comment by StripedMonkey** [comment]
I think I might have part of the issue, it's an MMS 🤔 when forwarding via the default text messenger it decided to forward it as an MMS.

**Comment by moezbhatti** [comment]
Ah, that was just the clue I needed!

Issue confirmed. Fixed in 7ac1971c5d564404583f658a97d6ba9e420debd2
